pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -303,6 +303,10 @@ func setReservation(disk *api.Disk) {
 }
 
 func setErrorPolicy(diskDevice *v1.Disk, disk *api.Disk) error {
+	if diskDevice.LUN != nil && diskDevice.LUN.Reservation && diskDevice.ErrorPolicy == nil {
+		disk.Driver.ErrorPolicy = v1.DiskErrorPolicyReport
+		return nil
+	}
```

Comment:
Several thought that come to my mind: do we really need to change the default behavior here? Is this change needed only for Windows Share Cluster Filesystem or is it a common requirement for the case with Persistent Reservation? TBH, I am not sure that we should handle each FS individually. Can't users of Windows Share Cluster Filesystem manually set the policy as needed?

Irrespective of the above concern, if we decide to go with this change, the API docstring will need an update to reflect it:

https://github.com/kubevirt/kubevirt/blob/99b5d4c79b8d990b92a62eb4b88fdb10b30f6105/staging/src/kubevirt.io/api/core/v1/schema.go#L647-L649

Reply:
The problem we are trying to solve is that in v1 we have the persistent reservation but we do not have the option to choose the behavior on the user side. My expectation is that here it is more likely to end up as a report policy than not. 
@vasiliy-ul @alicefr Do we know other use cases for reservation?

Reply:
@vasiliy-ul you are perfectly right! This is an alternative to this (ugly) backport: https://github.com/kubevirt/kubevirt/pull/10730
We already have the possibility to change the error policy in the release-1.1 and in the current main branch

Reply:
I think there is already a pending backport https://github.com/kubevirt/kubevirt/pull/10730.

I personally do not know much about usecases with reservation. Just to clarify: I am not against changing the default, but probably we need to take into account more FS types and not to base the decision just on a single one. But yeah, probably @alicefr knows more about that.

Reply:
> This is an alternative to this (ugly) backport: https://github.com/kubevirt/kubevirt/pull/10730

Why is it ugly? I think it is perfectly fine to backport it. By 'alternative', do you mean that the plan is to later backport this PR? That in fact would be a breaking change (i.e. this changes the default and documented behaviour).

Reply:
Ugly because I'm backporting an API change. The only use-case we have for SCSI persistent reservation is Windows Share Cluster Filesystem.
If we choose to merge this change, yes, I plan to backport this to 1.1 and 1.0. For 1.1, it applies without issues for 1.0, we need to manually backport, and I have it read [here](https://github.com/alicefr/kubevirt/commit/7b159774d6153f65f68a3d53582eb3ad7179c3a7)

Reply:
> Ugly because I'm backporting an API change. 

IMHO, backporting an API change is fine as long as it does not break compatibility and does not affect the existing behavior. In this case, you are just adding additional fields which will not do harm.

> If we choose to merge this change, yes, I plan to backport this to 1.1 and 1.0

Among the two, I would say that https://github.com/kubevirt/kubevirt/pull/10730 is a better approach since this PR changes the behavior. I.e. in general it is a breaking change.

Reply:
Yes, I agree. We are internally a bit short in time, that's why I have already opened this PRs to check the alternatives :).
@xpivarc wdyt?

Reply:
Would you review #10730 if you feel comfortable?

Reply:
I am with @vasiliy-ul on this, prefer #10730

Reply:
> Would you review https://github.com/kubevirt/kubevirt/pull/10730 if you feel comfortable?

Added my review. If we do not care about downgrades, I am ready to approve it. The code looks okay from my perspective.